File: romrepo.angular/src/app/model/system-setting.ts
export class SystemSetting {
    private name: string;
    private description: string;
    private value: string;
    private dataType: string;
    private isRequred: boolean;
    private isReadOnly: boolean;

    constructor(name: string, description: string, value: string, dataType: string, isRequired: boolean, isReadOnly: boolean) {
        this.name = name;
        this.description = description;
        this.value = value;
        this.dataType = dataType;
        this.isRequred = isRequired;
        this.isReadOnly = isReadOnly;
    }

    setValue(name:string, value:string) {
        
    }
}

File: romrepo.lib/Models/SystemSetting.cs
using Microsoft.EntityFrameworkCore;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace romrepo.lib.Models
{
    [Serializable]
    public class SystemSetting
    {
        [Key]
        public string Name { get; set; }
        public string Description { get; set; }
        public string? Value { get; set; }

        public string DataType { get; set; }
        public bool IsRequired { get; set; }
        public bool IsReadOnly { get; set; }
    }
}


